src/routes/tags.rs

```diff
@@ -20,17 +20,12 @@ pub fn config(cfg: &mut web::ServiceConfig) {
     );
 }
 
-// TODO: searching / filtering? Could be used to implement a live
-// searching category list
```

Comment:
Same with all of these;  Don't remove TODOs that aren't fixed yet.  This one may not be that useful, but you should justify how each of these are fixed or not needed.